engine/src/main/resources/org/terasology/include/globalFunctionsFragIncl.glsl

```diff
@@ -40,14 +40,8 @@ float calcLambLight(vec3 normal, vec3 lightVec) {
     return diffuse;
 }
 
-float calcSpecLight(vec3 normal, vec3 lightVec, vec3 eyeVec, float exp) {
-    vec3 halfWay = normalize(eyeVec+lightVec);
-    return pow(clamp(dot(halfWay, normal), 0.0, 1.0), exp);
-}
-
 float calcSpecLightNormalized(vec3 normal, vec3 lightVec, vec3 eyeVec, float exp) {
-    vec3 halfWay = normalize(eyeVec+lightVec);
-    return ((exp + 8.0) / PI_TIMES_8) * pow(clamp(dot(halfWay, normal), 0.0, 1.0), exp);
+    return pow(max(0.0, dot(-eyeVec, reflect(lightVec, normal))), exp);
```

Comment:
This change makes a less spreading specular light.  Although the original method had some promise,  it just didnt look correct as the cone of light coming off did not match the reflection on the wall.    This formula is based off http://www.tomdalling.com/blog/modern-opengl/07-more-lighting-ambient-specular-attenuation-gamma/

Before:
![terasology-160408105243-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394686/572217f8-fd82-11e5-8ee0-97901a850ec7.jpg)

After:
![terasology-160408105228-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394689/5b81f3e0-fd82-11e5-8d9f-56bac29d64b2.jpg)